client/src/App.js

```diff
@@ -20,7 +20,7 @@ const App = () => {
     email: "",
     password: "",
     _id: "609cb2b88951a63fccdcf790",
-    authId: "db3edf52-df87-45b9-9ea5-cd3a38cc2807",
+    authId: "aa94729e-866b-47b1-8c7a-ede2cfd25e51",
     loggedIn: false,
```

Comment:
Any reason for changing the default authId but not the _id?

Reply:
A new authID is generated on each login, and I think Chase was using the same user for testing out his login page.  User id stays constant however.